src/api/app/assets/stylesheets/webui2/list-group-item-integrated.scss

```diff
@@ -1,3 +1,7 @@
 .list-group .list-group-item-integrated:last-child {
     border-bottom: none;
-}
\ No newline at end of file
+}
+
+.no-list-style {
```

Comment:
This seems to be dropped again in a later commit. If that's the case please remove the change from this commit

Reply:
Removed